lib/generators/double_entry/install/templates/migration.rb

```diff
@@ -54,9 +54,21 @@ def self.up
       t.text       "log"
       t.timestamps                             :null => false
     end
+
+    create_table "double_entry_line_metadata", :force => true do |t|
+      t.integer    "line_id",               :null => false
+      t.string     "key",     :limit => 48, :null => false
+      t.string     "value",   :limit => 64, :null => false
+      t.timestamps                          :null => false
+    end
+
+    add_index "double_entry_line_metadata", ["line_id"], :name => "lines_meta_line_id_idx"
+    add_index "double_entry_line_metadata", ["key"],     :name => "lines_meta_key_idx"
+    add_index "double_entry_line_metadata", ["value"],   :name => "lines_meta_value_idx"
```

Comment:
If we did something like:
```
SELECT lines.* FROM lines
JOIN double_entry_metadata ON double_entry_metadata.line_id = lines.id
WHERE key = "color" AND value = "purple"
```
Would it use the `[line_id, key, value]` index or would it try to use an index on `line_id` first for the join and then an index on `key, value`?